website/docs/d/fabric_wide_settings.html.markdown

```diff
@@ -32,11 +32,17 @@ data "aci_fabric_wide_settings" "example" {}
 * `description` - (Optional) Description of object Fabric-Wide Settings Policy.
 * `name_alias` - (Optional) Name alias of object Fabric-Wide Settings Policy.
 * `disable_ep_dampening` - (Optional) Disable Ep Dampening knob of object Fabric-Wide Settings Policy. 
+* `domain_validation` - (Optional) Validate that static path is added but no domain is associated to an EPG.
```

Comment:
Validate that the correct physical domain is added before associating a new static path to an EPG.

Reply:
ok